lstchain/reco/utils.py

```diff
@@ -227,9 +235,16 @@ def camera_to_sky(pos_x, pos_y, focal, pointing_alt, pointing_az):
     focal = 28*u.m
     pointing_alt = np.array([1.0, 1.0]) * u.rad
     pointing_az = np.array([0.2, 0.5]) * u.rad
-    sky_coords = utils.camera_to_sky(pos_x, pos_y, focal, pointing_alt, pointing_az)
+    sky_coords = utils.camera_to_altaz_frame(pos_x, pos_y, focal, pointing_alt, pointing_az)
 
     """
+
+    if time:
```

Comment:
This if is not needed. You use `obstime=obstime` and it work fine also in the case where `obstime is None`